### ISSUE TITLE
rewrite signing and verify sigstore documentation

### ISSUE THREAD

**Issue by jonvnadelberg** [sigstore/docs#101]
**Description**

The current signing documentation is not organized to be easily read by users, and should be updated to make it more accessible.

**Comment by devilkiller-ag** [comment]
Hey, Can I work on it?

**Comment by Neilblaze** [comment]
> The current signing documentation is not organized to be easily read by users, and should be updated to make it more accessible.

@jonvnadelberg +1 for that. I'm willing to take a shot, do you have any suggestions on how it can be refactored? Thanks!

**Comment by jonvnadelberg** [comment]
@devilkiller-ag @Neilblaze I have some suggestions from engineering on it.   We can work on it together.  Some ideas:

- Change "Working with blobs" to "Signing with blobs"
- Signing with self-managed keys
- Signing registry images
- signing with containers
- verifying

Any other ideas?  How do we want to coordinate?

**Comment by devilkiller-ag** [comment]
Sure @jonvnadelberg, let's first figure out how to co-ordinate together and then let's start working on this.

**Comment by Neilblaze** [comment]
> Any other ideas? How do we want to coordinate?

@jonvnadelberg Let's do it! Feel free to hit me up on Slack (Pratyay Banerjee) or on [Discord](https://discordapp.com/users/694942064131047461) 🚀 

I had a small discussion with [Lisa](https://github.com/ltagliaferri) on another issue & as far as I can recall, I do have a bunch of ideas (some of which intersect with what you've mentioned) that we can implement to refactor the overall UX of the aforementioned.

**Comment by Queen-codes** [comment]
@jonvnadelberg  Hi. If there's a space for one more person, I'd love to work on this. I've been wanting to get the hang of collaborating with people on an issue. Thank you.

**Comment by jonvnadelberg** [comment]
Hi Everyone.  

I have specified five different topics that can be discussed and created as top-level subjects.   Does anyone have any additional topics they'd like to mention?

**Comment by Neilblaze** [comment]
I'm fine with both. I think it does need further discussion before we can start working on it.

**Comment by Queen-codes** [comment]
Hi. anyone's okay with me.

**Comment by jonvnadelberg** [comment]
OK, what I'm going to do is create the files I mentioned above.  I will edit the "working with blobs" file.   

There are also the following:

Signing with self-managed keys
Signing registry images
signing with containers
verifying

So please pick one of these and enter the content for it.  I will then review it as the tech writer.  Then we'll do a pull request

How's that sound?   Just pick one here and start editing for now.  If you have additional ideas, lets discuss.

I've assigned this issue to each of us.

**Comment by devilkiller-ag** [comment]
I will do the 'Signing with self managed key part '.

**Comment by Queen-codes** [comment]
@jonvnadelberg I'd do the signing registry images part

**Comment by haydentherapper** [comment]
Hey folks, please hold off on working on these yet. Before beginning work, we'll want to discuss the purposes of these rewrites and what we're looking to get out of them.

**Comment by jonvnadelberg** [comment]
Switch in plans:

we're holding off on doing multiperson work on this for right now.  I'm going to be doing the work on it.

**Comment by Neilblaze** [comment]
@jonvnadelberg I think you misinterpreted what @haydentherapper wanted to say 😅

**Comment by haydentherapper** [comment]
jonvnadelberg will be solely focusing on this issue going forward, as it's scoped for a single person.

**Comment by Neilblaze** [comment]
Ah okay, sounds good!

**Comment by haydentherapper** [comment]
Adding https://github.com/sigstore/docs/pull/116/files/d4f44bca0a589da62a6fdbf3364ddac72ccfea69#r1162107168 as a follow up

**Comment by jonvnadelberg** [comment]
changes have been written and merged in.
